politeiad/cache/cache.go

```diff
@@ -159,6 +159,11 @@ type Cache interface {
 	// Get the latest version of a record
 	Record(string) (*Record, error)
 
+	// Get the latest version of a record based on its prefix.
+	// The length of the prefix is defined by TokenPrefixLength in the
+	// politeiad api.
+	RecordPrefix(string) (*Record, error)
```

Comment:
Maybe `RecordByPrefix()` is a better name?  `Record()` and `RecordVersion()` and named by what is being returned where as `RecordPrefix()` is named by what is being passed in.

Thoughts?

Reply:
Yeah that sounds better for sure.